### PR TITLE
Webpack feature

### PR DESCRIPTION
**Pull request by Fsher07**
-Added a function to add new tasks
-Added a function remove tasks and update indexes of elements
-Added a function that provides users to edit tasks (must click on the task twice to be able to do that)
-Removed all hardcore items

### CODE REVIEW

File: src/taskOperations.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+  // static uncompleteTask = (index) => {
+  //   TaskOperations.tasks[index].completed = false;
+  //   localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+  // }
+
+  // static sortTask = () => {
+  //   TaskOperations.tasks.sort((a, b) => a.index - b.index);
+  //   localStorage.setItem('datas', JSON.stringify(TaskOperations.tasks));
+  // }
```
Comment (Lordkaito) [inline src/taskOperations.js:87]: - [ ] Please, make sure to delete all the commented code to try to keep your code clean.

File: src/taskOperations.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  //         if (e.target.parentElement.parentElement.checked) {
+  //           TaskOperations.completeTask(index);
+  //         } else {
+  //           TaskOperations.uncompleteTask(index);
+  //         }
+  //         TaskOperations.renderTasks();
+  //       }
+  //     }
+  //   });
+  // }
```
Comment (Lordkaito) [inline src/taskOperations.js:146]: - [ ] As I mentioned before, please try to avoid commented code, so please remove it from your codebase.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Fsher07, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Rustamxon7 — APPROVED** [review]
## Hello @Fsher07 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**